CHANGELOG.md

```diff
@@ -2,6 +2,40 @@
 
 ## Release 2.4.3 (in development)
 
+### Features added
+- [#1993](https://github.com/scality/metalk8s/issues/1993) - Add Solutions
+management, CLI tooling to deploy Solutions (complex Kubernetes applications)
+(PR [#2279](https://github.com/scality/metalk8s/pull/2279))
+
+### Enhancements
+- Add `label_selector` in MetalK8s custom kubernetes salt module for listing
+kubernetes objects (PR [#2236](https://github.com/scality/metalk8s/pull/2236))
+
+- [#2328](https://github.com/scality/metalk8s/issues/2328) - Bump K8S version
+to 1.15.11 (PR [#2362](https://github.com/scality/metalk8s/pull/2362))
+
+### Bug fixes
+- [#2334](https://github.com/scality/metalk8s/issues/2334) - Add and
+install `yum-utils` package required for cluster expansion
+(PR [#2343](https://github.com/scality/metalk8s/pull/2343))
+
+- [#2245](https://github.com/scality/metalk8s/issues/2245) - Rephrase volume
+status from `Available` to `Ready`
+(PR [#2248](https://github.com/scality/metalk8s/pull/2248))
+
+- Fix python invalid `time.sleep` argument
+(PR [#2257](https://github.com/scality/metalk8s/pull/2257))
+
+- Better handling of `wait_minions` when `test.ping` raises an exception
+(PR [#2308](https://github.com/scality/metalk8s/pull/2308))
+
+- Better handling of `wait_minions` when `test.ping` raises an exception
+(PR [#2308](https://github.com/scality/metalk8s/pull/2308))
```

Comment:
I think if we don't have documented bugs, we don't need an item in the Changelog. Those three are "internal details", I would get rid of them.

Reply:
Done